.github/workflows/ci.yml

```diff
@@ -20,14 +20,18 @@ jobs:
       - name: Noop
         run: "true"
   ci:
-    name: (${{ matrix.os }}) CI
```

Comment:
I removed this, as I could not make it look nice using `(${{ matrix.os }} ${{ matrix.target }}) CI`, as then there will be an space for runs that do not se `target`